src/utils.jl

```diff
@@ -18,6 +23,15 @@ text_only(doc::HTMLDocument) = text_only(doc.root)
 text_only(frag) = join([replace(text(leaf), "\r","") for leaf in Leaves(frag) if leaf isa HTMLText], " ")
 text_only(frags::Vector) = join(text_only.(frags), " ")
 
+function filter_html(random)
+    if random isa Void
+        return ""
+    end
+    if ismatch(r"<(\"[^\"]*\"|'[^']*'|[^'\">])*>", random)
```

Comment:
I like this idea, that you check if it possibly could be HTML, using regex, before parsing it and stripping tags.
It makes for a bit faster then parsing unnecessarily.
It would be good to add a comment to this line saying: "Check if might be HTML"
so it is clear that is what is going on.